target/linux/ath79/nand/base-files/etc/board.d/02_network

```diff
@@ -8,6 +8,11 @@ ath79_setup_interfaces()
 	local board="$1"
 
 	case "$board" in
+	glinet,gl-ar750s-nor |\
```

Comment:
Nitpick, but spaces before the "|\" are not common in ath79.

Reply:
_(Done)_